keyboards/era/divine/info.json

```diff
@@ -0,0 +1,115 @@
+{
```

Comment:
Please run `qmk format-json` on this file.

Reply:
> Please run `qmk format-json` on this file.

Sorry for the question.
What does it mean to run qmk format-json?

Reply:
Confirmed, Thank you sir.

Reply:
modified. Thanks for your help.